### ISSUE TITLE
gnostic-go-generator: -output doesn't like non package names

### ISSUE THREAD

**Issue by bhenderson** [googleapis/gnostic#111]
I know the docs say that -out is also the package name, but the error message is confusing. when I do something like `-output=/tmp/bookstore` I get

```
Plugin error: [constants.go:3:9: expected 'IDENT', found '/']
```
I think it would be nice to have a friendlier error.

I've traced this to goimports errors. related to #110 
https://github.com/googleapis/gnostic/blob/d55a06a32dc7468c645ac01d93d37d4dd47ca062/plugins/gnostic-go-generator/renderer.go#L63

I made some changes like

```
diff --git a/plugins/gnostic-go-generator/renderer.go b/plugins/gnostic-go-generator/renderer.go
index 93fd47d..31efb8e 100644
--- a/plugins/gnostic-go-generator/renderer.go
+++ b/plugins/gnostic-go-generator/renderer.go
@@ -60,7 +60,9 @@ func (renderer *Renderer) Render(response *plugins.Response, files []string) (er
                }
                // run generated Go files through imports pkg
                if filepath.Ext(file.Name) == ".go" {
-                       file.Data, err = imports.Process(file.Name, file.Data, nil)
+                       if file.Data, err = imports.Process(file.Name, file.Data, nil); err != nil {
+                               response.Errors = append(response.Errors, fmt.Sprintf("ERROR %v", err))
+                       }
                }
                response.Files = append(response.Files, file)
        }
```

The last error gets reported twice though due to named returns, and I was thinking it would be really nice to still write out the files so that the user can inspect the problem in context. 

thoughts?

**Comment by bhenderson** [comment]
The code I put above is really for better error messaging.

I'm working on a PR for cleaning the package name.

**Comment by bhenderson** [comment]
Closed by #112
